tests/unit/api/test_model.py

```diff
@@ -419,7 +419,7 @@ def test_validate_model_bad_args(client, building_motif):
     assert results.status_code == 400
 
 
-def test_test_model_against_shapes(client, building_motif, shacl_engine):
+def test_validate_model_against_shapes(client, building_motif, shacl_engine):
```

Comment:
Should this test be moved to `test_compiled_model.py` since its target def is in `compiled_model.py`?

Reply:
I'd like to keep the test here because it has to do with the /models API endpoint